src/signals/incident/components/form/MapSelectors/components/MapMessage/MapMessage.tsx

```diff
@@ -82,7 +86,7 @@ export const ZoomMessage: FC<PropsWithChildren<ZoomMessageProps>> = ({
 
 interface MapMessageProps
   extends PropsWithChildren<Omit<HTMLAttributes<HTMLDivElement>, 'onClick'>> {
-  onClick: MouseEventHandler<SVGElement>
+  onClick: () => void
```

Comment:
kan nog beschrijvender met MouseEventHandler<HTMLElement>

Reply:
Heb dit nu als type: MouseEventHandler<HTMLAnchorElement> & MouseEventHandler<HTMLButtonElement>
Hoewel het descriptive is, is het ook veel meer code / onleesbaarder. Op andere plekken in de code wordt vaker () => void gebruikt voor onClick.